contentgrid-spring-boot-actuators/src/main/java/com/contentgrid/spring/boot/actuator/policy/RegoProducible.java

```diff
@@ -0,0 +1,19 @@
+package com.contentgrid.spring.boot.actuator.policy;
+
+import org.springframework.boot.actuate.endpoint.Producible;
+import org.springframework.util.MimeType;
+
+public enum RegoProducible implements Producible<RegoProducible> {
```

Comment:
There is no need to have this as an enum, just a class would be okay?

Reply:
If I understand `interface Producible<E extends Enum<E> & Producible<E>>` correctly, it needs to be an enum.

From [org.springframework.boot.actuate.endpoint.Producible](https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/Producible.java)